src/time/span.cr

```diff
@@ -53,18 +53,21 @@ struct Time::Span
   # @nanoseconds can either be negative or positive).
   @nanoseconds : Int32
 
-  def self.new(hours : Int, minutes : Int, seconds : Int)
-    new(0, hours, minutes, seconds)
+  def self.new(days : Int, hours : Int, minutes : Int, seconds : Int, nanoseconds : Int)
```

Comment:
Do we even want to keep this overload? Why not require named arguments always?

Reply:
I was slightly worried about backwards compatability, but if this seems unnecessary, I will remove it

Reply:
We're breaking backwards compatibility anyway. But it might make sense to leave it for now as deprecated.